etl/steps/data/garden/wb/2023-12-29/gender_statistics.meta.yml

```diff
@@ -13,30 +13,68 @@ tables:
+          - This indicator uses standardized assumptions, like the woman having one child and residing in the largest business city, to ensure comparability, though this approach may not capture variations in laws affecting women in different states, rural areas, or minority groups
+
         processing_level: major
         display:
           numDecimalPlaces: 0
+      total_maternity_leave_for:
+        title: Total days of leave available for the mother
+        unit: days
+        short_unit: ''
+        description_processing: This indicator was created by OWID by combining maternity leave and parental leave mother quota indicators. For more details on each indicator see **How producer describes this data**.
+        description_key:
+          - This indicator is created by combining three specific types of parental leave. These components, together, provide a detailed understanding of the total leave available to mothers in relation to childbirth and early childcare.
+          - The first component, the length of paid maternity leave, focuses on the leave specifically designated for mothers at the time of childbirth. This includes the days available exclusively to mothers to support them in bonding with the newborn and assisting in early childcare. This leave plays a crucial role in supporting the mother's engagement during the early stages of a child's life.
+          - The second component, the length of paid parental leave for mother, includes days of parental leave that are specifically reserved for mothers. These are often defined by policies that support gender equality in parental responsibilities. This part of the indicator represents additional support provided to mothers, extending beyond the traditional maternity leave period.
+          - The third component, the length of shared parental leave, represents the portion of leave that can be shared between parents. This flexibility allows for a more equitable distribution of childcare responsibilities and supports gender equality.
+          - By integrating these components, the indicator provides a total count of the calendar days of leave available to mothers.
+          - This indicator uses standardized assumptions, like the woman having one child and residing in the largest business city, to ensure comparability, though this approach may not capture variations in laws affecting women in different states, rural areas, or minority groups
+
+        processing_level: major
+        display:
+          numDecimalPlaces: 0
+
+      total_paternity_leave_for:
+        title: Total days of leave available for the father
+        unit: days
+        short_unit: ''
+        description_processing: This indicator was created by OWID by combining paternity leave and parental leave father quota indicators. For more details on each indicator see **How producer describes this data**.
```

Comment:
... paternity leave, parental leave father quota and shared parental leave indicators...